packages/playwright-core/src/server/dispatchers/dispatcher.ts

```diff
@@ -357,7 +357,7 @@ export class DispatcherConnection {
     }
 
     if (response.error)
-      response.log = callMetadata.log;
+      response.log = compressCallLog(callMetadata.log);
     this.onmessage(response);
   }
```

Comment:
I expect the removal of `formatCallLog(log)` in `Connection::dispatch` since the server side ensures now that its formatted. The goal should be no client-side usages of `formatCallLog` anymore.

Reply:
dispatcher is server and it compresses (not formats)